### PR TITLE
Devise password redirects

### LINKED ISSUE #1327: [Bug]: Password reset redirects you to root url on success
## Description

After password reset, you get redirected to placecal.org

## Steps to reproduce

1. Click forgot pw
2. Fill in form
3. You're on placecal.org

## What you expected to happen

password reset should redirect you to the login page.

enter new password (accept invitation) should take you to the admin page.

## Implementation plan
- [ ] look at devise passwords controller and try to figure out how the redirect URL is generated
- [ ] make some tests covering this?

### PR DESCRIPTION
**Pull request by ivankocienski**
Fixes #1327 - send users to correct domains

### CODE REVIEW

**Review by alexandria-gfsc — CHANGES_REQUESTED** [review]
The facebook stuff is in the wrong branch :)

File: app/views/admin/users/profile.html.erb [hunk 1]
```diff
@@ -35,8 +35,6 @@
     <div class="col-md-6">
       <h2>Facebook integration</h2>
       <p>Only needed if you want to synchronise Facebook events with this account. <a href="https://handbook.placecal.org/how-to-use-placecal/admins/facebook">See the handbook page for how to do this</a>.</p>
-      <%= f.input :facebook_app_id, as: :string %>
-      <%= f.input :facebook_app_secret, as: :string %>
```
[the remark this answers is not in the document]
Comment (ivankocienski) [reply app/views/admin/users/profile.html.erb:39]: lol that will be fun to merge

File: test/support/email_helper.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  end
+
+  # extract a link from an email
+  #
+  # Parameters
+  #   email: the email returned by `last_email_delivered`
+  #
+  # Returns
+  #   string of link if found or nil
+  def extract_link_from(email)
```
Comment (alexandria-gfsc) [inline test/support/email_helper.rb:18]: Maybe change to "extract first link from email"? tiny nitpick and im fine to merge without this though

**Review by alexandria-gfsc — APPROVED** [review]
LGTM

File: test/support/email_helper.rb (hunk 2 above)
Comment (ivankocienski) [reply test/support/email_helper.rb:18]: ah fair point. it used to be as you suggest but aesthetically it reads like english: `extract_link_from(email)` as opposed to `extract_link_from_email(email)`. i believe this is supposed to be "self documenting code" but i can make it explicit

**Pull request merged**